### ISSUE TITLE
Error window on start: Unable to load NFS_XtendedInput.asi. Error: 126 

### ISSUE THREAD

**Issue by Szab0t** [xan1242/NFSU-XtendedInput#28]
So this was working initially but few days later just stopped working on all NFS version. I have other .asi file loaded parallel with this and those ones still loading fine (like HD patch). Game start up after this error message however without xinput support.

**Comment by xan1242** [comment]
Error 126 means that a module is missing (DLL missing).

You need to install the VCRedist: https://aka.ms/vs/17/release/vc_redist.x86.exe

And if you're running Win7 install this (whichever is applicable to your game, UG or UG2):

https://github.com/xan1242/NFSU-XtendedInput/releases/download/2.7/Release-UG-Win7-Pack.zip
https://github.com/xan1242/NFSU-XtendedInput/releases/download/2.7/Release-UG2-Win7-Pack.zip

**Comment by Szab0t** [comment]
Yes looks like something corrupted my VCRedist installation. Thanks for the link. You can close this issue now. Thanks again.

**Comment by rtx408016gb** [comment]
thanks finally **nfshp2** is running 2560 1440 60fps
